src/components/AdminPanel/AdminPanel.css

```diff
@@ -0,0 +1,18 @@
+.admin-panel {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+}
+
+.form-option {
+    cursor: pointer;
+    padding: 8px;
+    display: inline-block;
+}
+
+.form-option:hover {
+    color: #34db3c;
+}
```

Comment:
Rzecz kosmetyczna. Kuba podkreślał, żeby utrzymywać kod w czystości. Jak dajemy enter pomiędzy elementami to wszędzie. Coś na zasadzie jak moje 'import React' :)